.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -1,39 +1,31 @@
 ---
 name: "\U0001F41E Bug report"
 about: Report a bug if something isn't working as expected in the WooCommerce iOS app.
-
+title: ""
+labels: "[Type] Bug"
+assignees: ""
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is. Please be as descriptive as possible; issues lacking detail, or for any other reason than to report a bug, may be closed without action.
+### Expected behavior
+
+
+### Actual behavior
 
-**To Reproduce**
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
 
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
+### Steps to reproduce the behavior
 
-**Expected behavior**
-A clear and concise description of what you expected to happen.
 
-**Isolating the problem (mark completed items with an [x]):**
+##### Tested on [device], iOS [version], WooCommerce app [version]

```

Comment:
I feel like the previous **Mobile Environment** section is easier to _fill in_ than this. This requires replacing the placeholders like `[device]` while the previous one only requires the user to enter the new information. So probably faster and easier to understand? 

Also, the `####` is a [header in Markdown](https://help.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#headings) so it kinda looks odd that it doesn't have content. 🙂 

```suggestion
#### Mobile Environment

Please include:

- Device: 
- iOS version: 
- WooCommerce app version:
```

☝️ I'm not sure if we need to include the `"Please include"` text though. What do you think, @designsimply?

Reply:
I'm all for making it easier to fill in! This update looks good to me, thanks for adding it!